app/src/CoursePages/CoursePage.css

```diff
@@ -7,17 +7,141 @@
   right: 0%;
   margin: 3%;
   gap: 1rem;
+  overflow: scroll;
+  justify-content: space-around;
+}
+
+.ratingbox {
+  font-weight: bold;
+  font-size: clamp(1px, 130%, 3vw);
+  display: flex;
+  align-items: center;
+  align-self: flex-end;
+  justify-content: center;
+  height: 15vh;
+  max-height: 4ch;
+  min-width: 3ch;
+  width: 8vw;
+}
+
+.ratingbox-Difficulty {
+  background-color: #e087e2;
+}
+
+.ratingbox-Workload {
+  background-color: #8dbbff;
+}
+
+.ratingbox-Practicality {
+  background-color: #88ecaa;
 }
 
+/*
+  CSS styling for the top portion of the course page
```

Comment:
Maybe put the comment in 1 line (unless style guide says otherwise)?

Reply:
Fixed. Also moved CSS around and changed comments to create clearer sections.